addons/microsoft_calendar/models/microsoft_sync.py

```diff
@@ -42,9 +42,8 @@ def called_after():
                 env = api.Environment(cr, uid, context)
                 try:
                     func(self.with_env(env), *args, **kwargs)
-                except Exception as e:
-                    _logger.warning("Could not sync record now: %s" % self)
-                    _logger.exception(e)
+                except Exception:
```

Comment:
ci/style failed because of `except Exception`

```suggestion
                except TypeError:
```

Reply:
@abz-odoo  done